pyomo/common/tests/test_config.py

```diff
@@ -1479,12 +1479,8 @@ def test_block_keys(self):
         keys = self.config['scenario'].keys()
         # lists are independent
         self.assertFalse(keys is self.config['scenario'].keys())
-        if PY3:
-            self.assertIsNot(type(keys), list)
-            self.assertEqual(list(keys), ref)
-        else:
-            self.assertIs(type(keys), list)
-            self.assertEqual(keys, ref)
+        self.assertIs(type(keys), list)
```

Comment:
Should this be `assertIsNot`? Seems like you switched this from the old PY3 assertion (same with lines 1504 and 1522)